src/server/server.ts

```diff
@@ -121,6 +140,11 @@ export class CTProtoServer<AuthRequestPayload, AuthData, ApiRequest extends NewM
       let msgWaiter: NodeJS.Timeout;
       const msgWaitingTime = 3000;
 
+      /**
+       * Set type of incoming binary data to Buffer, because default value is blob
+       */
+      socket.binaryType = 'nodebuffer';
```

Comment:
still can't get it. It works fine in regular messages. Why do we need this? JSDoc is not clearly answers on that question